plugin.xml

```diff
@@ -30,7 +30,7 @@ xmlns:android="http://schemas.android.com/apk/res/android"
     <issue>https://issues.apache.org/jira/browse/CB/component/12320651</issue>
 
     <engines>
-        <engine name="cordova-android" version=">=6.3.0" />
+        <engine name="cordova-android" version=">=6.0.0" />
```

Comment:
Why this change?

Reply:
In fact we have still build configurations with cordova-android: 6.0.0
We haven't recognized an incompatibility with cordova-plugin-file. Was there a serious reason behind to restrict a higher cordova-android version?